lib/opera/operation/instructions/executors/transaction.rb

```diff
@@ -18,6 +17,12 @@ def call(instruction)
             nil
           end
 
+          def transaction_wrapper
+            return transaction_class.send(transaction_method) { yield } unless transaction_options
+
+            transaction_class.send(transaction_method, **transaction_options) { yield }
+          end
+
```

Comment:
I wonder… can we simplify it to: https://github.com/Profinda/opera/compare/ENG-708-ruby-3...mike/ruby-3

Specs are green but I don't know the internals well enough 🙈 
@petergebala ? 🙂

Reply:
Hah.. It was my [initial implementation](https://github.com/Profinda/opera/pull/22/commits/42319d89fc8d744800303001db6893af527fc8df#diff-1767667f0614e3bd42ca16f1e470763b296d8a2cbf27ae17430b6cb4fec6687fR11). The thing is the r[elated specs fail](https://github.com/Profinda/opera/actions/runs/6076307424/job/16484072241) for ruby 2.6, so I switched to this solution.

Reply:
Hah… missed it but it makes perfect sense 😄